docs/application/vstools/tools/certificate-manager.md

```diff
@@ -1,104 +1,254 @@
 
-        ![Use default certificates](media/vstools_cert_default.png)
+       To use the default certificates, clear the **Sign the .TPK file using the following option.** checkbox.
 
-    -   **Using an existing certificate profile**
+       ![Use default certificates](media/vstools_cert_default.png)
 
-        If you have used Tizen Studio before and have already generated a certificate profile using the Tizen Certificate Manager, you can import the profile by selecting **Use profile of Tizen Certificate Manager** from the drop-down list.
+    -  **Using an existing certificate profile**
 
-        If you want to create a new certificate profile, see [Creating a Certificate Profile](#creating).
+       If you have used Tizen Studio before and have already generated a certificate profile using the Tizen Certificate Manager, you can import the profile by selecting **Use profile of Tizen Certificate Manager** from the drop-down list.
 
-        ![Use existing certificate profile](media/vstools_cert_profile1.png)
+       ![Use existing certificate profile](media/vstools_cert_profile1.png)
 
-    -   **Using your own certificates**
+    -  **Using your own certificates**
 
-        If you already have author and distributor certificates from another application store, you can import them by selecting **Direct registration** from the drop-down list and entering the required information.
+       If you already have the author and the distributor certificates from some other application store, you can import them by selecting **Direct registration** from the drop-down list and entering the required information.
 
-        ![Use own certificates](media/vstools_cert_certificate1.png)
+       ![Use own certificates](media/vstools_cert_certificate1.png)
 
 3.  Click **OK**.
 
-<a name="creating"></a>
-## Creating a Certificate Profile
+## Create Certificate Profile
 
->**Note**
->
->It is recommended to keep your certificates and password safe in the local repo to prevent it from being compromised.
+If you already have a certificate profile, you can import it or you can create a certificate profile.
 
-A certificate profile consists of an author certificate and 1 or 2 distributor certificates:
+### Import certificate profile
 
--   An author certificate includes information about the author of the application. It is used to create an author signature, which ensures the integrity of the application from the author since the publication of the application.
--   A distributor certificate includes information about the distributor of the application, such as a store. It is used to create a distributor signature, which ensures the integrity of the application from the distributor since the distribution of the application.
+1. In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Certificate Manager**.
+2. To import the profiles.xml file, click **Browse** . 
```

Comment:
There i s an extra space after the word "Browse".
In the next commit, I will fix this.